examples/stm32/f4/stm32f4-discovery/usb_cdcacm/cdcacm.c

```diff
@@ -182,27 +190,28 @@ static int cdcacm_control_request(usbd_device *usbd_dev,
 		 * even though it's optional in the CDC spec, and we don't
 		 * advertise it in the ACM functional descriptor.
 		 */
-		return 1;
+		return USBD_REQ_HANDLED;
 		}
 	case USB_CDC_REQ_SET_LINE_CODING:
 		if (*len < sizeof(struct usb_cdc_line_coding)) {
-			return 0;
+			return USBD_REQ_NOTSUPP;
 		}
 
-		return 1;
+		return USBD_REQ_HANDLED;
 	}
-	return 0;
+	return USBD_REQ_NOTSUPP;
```

Comment:
These changes are great, and I actually though thtye'd already been done.  However, they need to be applied to all the examples